addons/website_theme_install/static/src/less/website_theme_install.less

```diff
@@ -1,42 +1,142 @@
+@o-theme-kanban-gray: #fcfcfc;
+
 .o_kanban_view.o_theme_kanban {
+    background-color: @o-theme-kanban-gray;
+
+    /// Un-grouped Layout (default)
     &.o_kanban_ungrouped {
-        .o-justify-content(center);
+        .o-justify-content(space-between);
+        margin: 0 -@grid-gutter-width;
```

Comment:
Mmh ^^ I don't know. I am not against negative margins but I prefer when it is clear why they are used. Here you put ml32/mr32 classes in the xml, which happen to be equal to `@grid-gutter-width` maybe it would be clearer if those horizontal margins where put in the LESS file for this case.

Anyway, the real problem here is that these negative margins are breaking xs/sm layout here, could you check? (and yes kanban view LESS is a mess, I will be happy to break all of that in a kanbanpocalypse some day :p). Maybe just removing these negative margins and put smaller ml/mr should be ok and safer ?